src/main/resources/application-dev.yaml

```diff
@@ -14,3 +14,22 @@
     exclude:
       - "org.springframework.boot.actuate.autoconfigure.security.servlet.ManagementWebSecurityAutoConfiguration"
       - "org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration"
+  security:
```

Comment:
These properties in `dev` and `stg` profiles can be removed if they are not overriding the properties specified in the default profile `src/main/resources/application.yaml`

Reply:
They are added as a placeholder for now.